WordPress/WordPressTest/MockWordPressOrgXMLRPCApi.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+@objc class MockWordPressOrgXMLRPCApi: NSObject, WordPressOrgXMLRPC {
+
+    var methodPassedIn: String? = nil
+    var parametersPassedIn: [AnyObject]? = nil
```

Comment:
Hmm, I was looking at `MockWordPressComRestApi` and I'm starting to think the mock approach is good for the XMLRPC tests but not for the REST tests. `MockWordPressComRestApi` does what `OHHTTPStubs` does (replacing network calls), only much simpler. So I'd rather use the existing library for that.

`MockWordPressOrgXMLRPCApi` however mocks the XMLRPC methods and avoids all the XML conversion and hitting the network layer, so I think that's interesting and more like I'd like our network architecture/testing to be in the long term. 

I don't like that we have to call the success/failure block manually or that it only keeps the last method/parameters, but it might be OK for a first PR and we can iterate on it later.